Breeze.BreezeServer/BreezeConfiguration.cs

```diff
@@ -82,6 +82,21 @@ public BreezeConfiguration(string configPath, string datadir = null)
                 {
                     TumblerNetwork = Network.Main;
                 }
+
+                if (configFile.GetOrDefault<string>("network", "testnet").Equals("stratistest"))
+                {
+                    TumblerNetwork = Network.StratisTest;
+                }
+
+                if (configFile.GetOrDefault<string>("network", "testnet").Equals("stratisregtest"))
+                {
+                    TumblerNetwork = Network.StratisRegTest;
+                }
+
+                if (configFile.GetOrDefault<string>("network", "testnet").Equals("stratismain"))
+                {
+                    TumblerNetwork = Network.StratisMain;
+                }

```

Comment:
Shouldn't this be converted from 3 `if`s to `if->else if -> else if` or for a cleaner code to a `switch` statement. There is no need to test all conditions if one of the succeeds